crates/store/re_log_types/src/index/non_min_i64.rs

```diff
@@ -208,6 +208,24 @@ impl<'de> serde::Deserialize<'de> for NonMinI64 {
     }
 }
 
+#[derive(thiserror::Error, Debug)]
+#[error("Failed to parse NonMinI64: {0}")]
+pub enum ParseNonMinI64Error {
+    Std(#[from] std::num::ParseIntError),
+
+    #[error("out-of-range")]
+    OutOfRange,
```

Comment:
out of range seems a bit odd. the range is really "everything but i64::MIN", so it's more like

```suggestion
    #[error("Value is equal to minimum i64. Every i64 integer *except* the lowest representable number of a signed 64 bit number is valid.")]
    InvalidValue,
```